panic: exec: "nspBuild.exe": cannot run executable found relative to current directory

I built the latest `main` on Windows 11 and am trying to use it on an fg release. Using the `nspBuild.exe` that came bundled with fitgirl-repacks-nsp, I'm getting:

```
Fetching file 03d7d225d3263914ecc615ce7bf1dc65.cnmt.nca
Fetching file 171f45e515f825bd3a8f3b9f1045d0a0.nca
Fetching file 1c996df722cccfde9b6272225c1a6239.cnmt.nca
Fetching file 39c65e81caf78a55b33036b814adee99.nca
Fetching file 3d11dfabd44b8da0c63e2960be899989.nca
Fetching file 40fbbe7f6204d10d82ce71c614037032.cnmt.nca
Fetching file 429e6f59ac05452637dc325969d0a678.nca
Fetching file 4909fa3c3c0093fc3620279834a7d8e5.cnmt.nca
Fetching file 7152b01a3015c92fb43b8896cf64dbd9.nca
Fetching file 8f09aa7dd095b0fbc18ff54ab86e5597.nca
Fetching file a6cbf31b496e0adb090f7d2b65cfae15.cnmt.nca
Fetching file af248f2ded494aa23160ca1d556bddad.nca
Fetching file cbce1a573b0d5b20dc0f3d9a49f97def.cnmt.nca
Fetching file f38e66d3d0f5ccec9249daa1cdfaccfe.nca
panic: exec: "nspBuild.exe": cannot run executable found relative to current directory

goroutine 1 [running]:
main.main()
        C:/Users/apetresc/src/personal/fitgirl-repacks-nsp/extractor.go:77 +0x2df
```

Which is weird, because it _did_ copy `nspBuild.exe` over to the release directory, and it does appear to be executable:

```
Usage is: C:\Users\apetresc\[gamedir]\nspBuild.exe output file1 file2 ...
```

--- a/extractor.go
+++ b/extractor.go
@@ -14,7 +14,7 @@ import (
 //go:embed nspBuild.exe
 var binary []byte
 
-const nspBuildExecutable = "nspBuild.exe"
+const nspBuildExecutable = "./nspBuild.exe"
 
 func extractNcaFilesPath(directoryPath string) []string {
 	filesPath := make([]string, 0)

